### PR TITLE
Add Launch Heavy

### PR DESCRIPTION
**Pull request by thomas-zimmerman**
Add Launch Heavy and slight updates around firmware updates.

### CODE REVIEW

**Review by leviport — APPROVED** [review]
Looks good, thanks!

**Review by ahoneybun — APPROVED** [review]
LGTM :taco:

File: content/launch-keyboard.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ In the <u>Keyboard Configurator</u>, you can change keyboard back light color an
 
 We have documentation that goes over changing the keycaps, switches, and more on our Tech Docs site:
 
-- [Launch](https://tech-docs.system76.com/models/launch_1/repairs.html)
+- [Launch launch_1](https://tech-docs.system76.com/models/launch_1/repairs.html)
+- [Launch launch_2](https://tech-docs.system76.com/models/launch_2/repairs.html)
```
Comment (leviport) [inline content/launch-keyboard.md:69]: "Launch launch_1" seems weird. Maybe should be "Launch (launch_1)" or just "launch_1"? If the latter, we probably want to change "Launch Lite" to "launch_lite_1" and "Launch Heavy" to "launch_heavy_1"

Comment (jacobgkau) [reply content/launch-keyboard.md]: I put them in parenthesis and added them for the other models with 6db34ba. This matches the formatting used in the Tech Docs sidebar (other than the order.)

image

Comment (leviport) [reply content/launch-keyboard.md]: Looks good!

**Review by leviport — APPROVED** [review]

**Review by ahoneybun — APPROVED** [review]

**Pull request merged**